pkg/controller/endpointslice/reconciler.go

```diff
@@ -154,8 +152,7 @@ func (r *reconciler) reconcileByAddressType(service *corev1.Service, pods []*cor
 	desiredEndpointsByPortMap := map[endpointutil.PortMapKey]endpointsliceutil.EndpointSet{}
 
 	for _, pod := range pods {
-		includeTerminating := service.Spec.PublishNotReadyAddresses || utilfeature.DefaultFeatureGate.Enabled(features.EndpointSliceTerminatingCondition)
-		if !endpointutil.ShouldPodBeInEndpoints(pod, includeTerminating) {
+		if !endpointutil.ShouldPodBeInEndpoints(pod, true) {
```

Comment:
Do we ever pass `false` here anymore? If not, could we remove the second param?

Reply:
I thought it was still used for Endpoints, but let me double check